Pod/Classes/VOKMappableModel.h

```diff
@@ -0,0 +1,32 @@
+//
+//  VOKMappableModel.h
+//  VOKCoreData
+//
+
+#import "VOKCoreDataManager.h"
+
+/**
+ *  Any models that conform to this protocol will be automatically registered for mapping with the shared instance 
```

Comment:
Has anyone ever used more than one instance of VOKCoreDataManager?
I don't think it's a problem with the code. Though this docu-comment could be more specific.